### PR TITLE
Serve built docs within app

### LINKED ISSUE #3803: Serve html docs if they exist
If the html docs exist at a particular location, then we'll serve them. This is so that built docs can be bundled with the app.

### PR DESCRIPTION
**Pull request by MCGallaspy**
Fixes #3803. Supersedes #3806.

Serves the docs statically, if they exist. Adds a conditional item to the user dropdown menu, with a link to the statically served docs. Adds an new mapping to the status model.

### CODE REVIEW

File: kalite/facility/api_resources.py [hunk 1]
```diff
@@ -177,6 +178,8 @@ def generate_status(self, request, **kwargs):
             "version": version.VERSION,
             "facilities": request.session.get("facilities"),
             "simplified_login": settings.SIMPLIFIED_LOGIN,
+            "docs_exist": True if any(map(lambda x: os.path.join("sphinx-docs", "_build") in x,
```
Comment (aronasorman) [inline kalite/facility/api_resources.py]: I believe you can remove the `if` statement and just use `any`, which returns a boolean.

Comment (MCGallaspy) [inline kalite/facility/api_resources.py]: You are correct.

**Comment by aronasorman** [comment]
Looks good to me. I'm waiting for the develop tests to get fixed. Once that's done then we can merge this.

**Comment by aronasorman** [comment]
Tests pass. Merging.

**Pull request merged**